Compilation of master fails it https is disabled.

##### Bug report summary
Compilation fails if https is disabled (e.g. via --enable-https=no).

```
exporting/send_data.c: In function ‘simple_connector_worker’:
exporting/send_data.c:262:14: error: ‘connector_specific_data’ undeclared (first use in this function); did you mean ‘connector_specific_config’?
         if (!connector_specific_data->first_buffer->used || failures) {
              ^~~~~~~~~~~~~~~~~~~~~~~
              connector_specific_config
exporting/send_data.c:262:14: note: each undeclared identifier is reported only once for each function it appears in

```

##### OS / Environment
Linux

##### Netdata version
git master 

##### Component Name
core

##### Steps To Reproduce
1. ./configure --enable-https=no
2. make

##### Expected behavior
Build works :-).

--- a/exporting/send_data.c
+++ b/exporting/send_data.c
@@ -236,10 +236,10 @@ void simple_connector_send_buffer(
 void simple_connector_worker(void *instance_p)
 {
     struct instance *instance = (struct instance*)instance_p;
+    struct simple_connector_data *connector_specific_data = instance->connector_specific_data;
 
 #ifdef ENABLE_HTTPS
     uint32_t options = (uint32_t)instance->config.options;
-    struct simple_connector_data *connector_specific_data = instance->connector_specific_data;
 
     if (options & EXPORTING_OPTION_USE_TLS)
         ERR_clear_error();

